subset/bacnet/bacnetTests/src/main/java/helper/FileManager.java

```diff
@@ -0,0 +1,64 @@
+package helper;
+
+import java.io.File;
+
+public class FileManager {
+
+    private String filePath = "";
+    private String csvName = "pics";
+    private String csvExtension = ".csv";
+    private boolean debug = false;
+
+    public boolean checkDevicePicCSV() {
+        String csvFolder = getCSVPath();
+        try{
+        File[] listFiles = new File(csvFolder).listFiles();
+        for (int i = 0; i < listFiles.length; i++) {
+            if (listFiles[i].isFile()) {
+                String fileName = listFiles[i].getName();
+                if (fileName.contains(csvName)
+                        && fileName.endsWith(csvExtension)) {
+                    System.out.println("pics.csv file found.");
```

Comment:
This is the last thing I can see - can we print the path where the pics.csv file wasn't found? It makes debugging a lot easier for someone who isn't familiar with the system (i.e. all current users)